### DISCUSSION TITLE
Troublesshooting: Build Issues

### DISCUSSION THREAD

**Discussion by KangChenRui** [idaholab/moose#22411] [Q&A Getting Started] [answered]
hello everyone, I have some problems.
After the moose is installed, some errors occurs when I "make - j 4".  Plaese see the below picture: 
![image](https://user-images.githubusercontent.com/91399933/196026577-a153f379-9b05-4478-9f47-2167acf04668.png)

Then, I found answer in the Troublesshooting: Build Issues, when I "git fetch upstream",the fatals is :
![image](https://user-images.githubusercontent.com/91399933/196026901-61067da8-feb1-469c-89a9-adb3299638fc.png)
Now, what should I do?
Thanks for your help！

**Comment by GiudGiud** [comment, marked as answer]
Hello

your upstream is probably called origin because you did not rename it. 
so type git fetch origin instead

Guillaume

**Reply by KangChenRui** [reply]
hello,Thanks for your help!
 Now,I encountered a new problem, as shown in the figure below：
![image](https://user-images.githubusercontent.com/91399933/196074226-683dadad-a5d8-4508-b1e2-d1fca2701fbe.png)

**Reply by GiudGiud** [reply]
you dont need to init submodules. large_media is only for building the documentation locally and getting the pictures

**Reply by KangChenRui** [reply]
However, compilation errors are consistent, as shown in the following figure：
![image](https://user-images.githubusercontent.com/91399933/196074818-46444a72-edab-41eb-8911-458ed01ace4d.png)

**Reply by GiudGiud** [reply]
what s the commit number?
are you using the conda installation path?
what is the output of the diagnostics script in moose/scripts

**Reply by KangChenRui** [reply]
About commit number, When I type “make -j 4”, errors are reported.
The installation path is completely in accordance with the installation guidelines
There is no "nohup.out" file.

**Reply by GiudGiud** [reply]
the commit number is the output of `git show` not make
which guidelines? there are more than one way to install the code
it's not the nohup file I'd like to see, please run the `diagnostics.sh` script in moose/scripts

**Reply by KangChenRui** [reply]
About guidelines,Refer to the following link “https://mooseframework.inl.gov/getting_started/installation/conda.html”.
The files generated after running diagnostics.sh are as follows：
[nohup.txt](https://github.com/idaholab/moose/files/9796637/nohup.txt)

**Reply by GiudGiud** [reply]
You have another compiler installed that is hiding the compilers we are installing with conda
``
/home/yydeng/intel/compilers_and_libraries_2019.5.281/linux/mpi/intel64/bin/mpicc
``

Are you setting environment variables to find that compiler? Did you add it to your PATH for example?
You need to let the CC, FC, CXX environment variables point to the compilers installed by conda as well
